### ISSUE TITLE
cannot view thumbnail or play video in encrypted group

### ISSUE THREAD

**Issue by ykktrcb7** [vector-im/element-android#3107]
#### Describe the bug
Similar to to https://github.com/matrix-org/matrix-react-sdk/pull/5826

#### To Reproduce
1. Receive video in encrypted group
2. Preview shows white/black (depending on theme used)
3. Cannot play video

#### Expected behavior
Preview should show thumbnail of video. upon clicking on video, the video should be played.

#### Screenshots
![video fail](https://user-images.githubusercontent.com/81748597/113473812-403c3b00-946c-11eb-86b7-14382683723c.jpg)

#### Smartphone (please complete the following information):
 - Device: huawei mate 20 x
 - OS: Android 10

#### Additional context
 - 1.1.0 - F-Droid
 - Homeserver: local instance

**Comment by tulir** [comment]
Clicking the play button seems to work for me already

**Comment by T-bond** [comment]
It was already a fix for it in 1.1.2.

**Comment by ykktrcb7** [comment]
I tried now v1.1.3. Play functionality is working, but still no thumbnail

**Comment by DoM1niC** [comment]
> I tried now v1.1.3. Play functionality is working, but still no thumbnail

yes still there on latest SourceCode and Videoplayback is already brick again :(
